VideoLocker/src/main/java/org/edx/mobile/view/adapters/CourseDiscussionResponsesAdapter.java

```diff
@@ -219,10 +219,11 @@ private void bindViewHolderToResponseRow(DiscussionResponseViewHolder holder, fi
             @Override
             public void onClick(View v) {
                 if (comment.getChildren().isEmpty()) {
-                    router.showCourseDiscussionAddComment(context, comment);
+                    if (!discussionThread.isClosed())
+                        router.showCourseDiscussionAddComment(context, comment);
```

Comment:
Thats something we actually want in this case, the `TalkBack` feature will focus on this view and just say that `Comments are closed for this response`.